src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -28,10 +38,32 @@ public void findAllBusinesses() {
     businesses.add(new Listing("The Shiny Bean Coffee & Tea", new MapLocation(40.496328, -79.944862), 4.9, "https://goo.gl/maps/AYH2QCL7pkoMBxHA8", "https://theshinybean.com/"));
     businesses.add(new Listing("Weisshouse", new MapLocation(40.456684, -79.925499), 4.3, "https://goo.gl/maps/7tuXn7QF2hh7ioGYA", "https://www.weisshouse.com/"));
   }
+  
+  // Is only here until the places API is implemented,
+  // and for test file to be able to set any static business LinkedList we want
+  // to try to use
+  public void setAllBusinesses(List<Listing> allBusinesses) {
+   businesses = allBusinesses;
+  }
```

Comment:
What will change once the API is implemented?

Reply:
Not sure yet, currently I do not know if the places api can be tested locally and even so, we have to somehow set all the returned businesses to the class's object variable. So I do not think this will be changed anytime soon

Reply:
Then is the bit in the comment about the Places API misleading? Should it be removed?